.project

```diff
@@ -1,17 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>retorch-st-eShopContainers</name>
+	<name>samples-giis-template-m1</name>
```

Comment:
I changed the project name to retorch-st-eShopContainers but now it has been changed back to the old value